packages/svelte/tests/AreaChart.test.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+  import { AreaChart } from "../types";
+  import type { AreaChart as AC } from "@carbon/charts";
+  import { ScaleTypes } from "@carbon/charts/interfaces";
+
+  let chart: AC = null;
+  let ref = null;
+
+  const data = [
+    { group: "Dataset 1", date: "2019-01-01T08:00:00.000Z", value: 0 },
+    { group: "Dataset 1", date: "2019-01-06T08:00:00.000Z", value: -37312 },
+    { group: "Dataset 1", date: "2019-01-08T08:00:00.000Z", value: -22392 },
+    { group: "Dataset 1", date: "2019-01-15T08:00:00.000Z", value: -52576 },
+    { group: "Dataset 1", date: "2019-01-19T08:00:00.000Z", value: 20135 },
+    { group: "Dataset 2", date: "2019-01-01T08:00:00.000Z", value: 47263 },
+    { group: "Dataset 2", date: "2019-01-05T08:00:00.000Z", value: 14178 },
+    { group: "Dataset 2", date: "2019-01-08T08:00:00.000Z", value: 23094 },
+    { group: "Dataset 2", date: "2019-01-13T08:00:00.000Z", value: 45281 },
+    { group: "Dataset 2", date: "2019-01-19T08:00:00.000Z", value: -63954 },
+  ];
```

Comment:
is there anyway that these demo data pieces can just be imported from `@carbon/charts/demo/data` so we won't have to maintain them separately? and possibly the options?

Reply:
- updated the test files to use fixture data from `@carbon/charts/demo/data`
- casting demo options to `any` to avoid enum errors